src/runtime/offers-flow.ts

```diff
@@ -23,6 +23,7 @@ import {
   SubscribeResponse,
   ViewSubscriptionsResponse,
 } from '../proto/api_messages';
+import {ExperimentFlags} from './experiment-flags';
```

Comment:
Changes to this file need tests.

Reply:
I was just validating which method to use for the experiment for this control